x-pack/plugins/monitoring/public/application/pages/access_denied/index.tsx

```diff
@@ -62,6 +62,12 @@ export const AccessDeniedPage: React.FC<ComponentProps> = () => {
         the monitoring cluster."
           />
         </p>
+        <p>
+          <FormattedMessage
+            id="xpack.monitoring.accessDenied.noRemoteClusterClientDescription"
+            defaultMessage="If the setting `monitoring.ui.ccs.enabled` is set to `true`, make sure your cluster has the `remote_cluster_client` role on at least one node."
```

Comment:
My first thought here is "wouldn't kibana already know if `monitoring.ui.ccs.enabled` is true?"

Reply:
Yes, I just got a little lazy 😊  I'll pull in the config on that page and only show this if it's true!